diff --git a/queries/markdown/highlights.scm b/queries/markdown/highlights.scm
index 5d77e721..0b2ebd3e 100644
--- a/queries/markdown/highlights.scm
+++ b/queries/markdown/highlights.scm
@@ -1,34 +1,34 @@
 ;From MDeiml/tree-sitter-markdown & Helix
 (setext_heading
   (paragraph) @markup.heading.1
-  (setext_h1_underline) @markup.heading.1)
+  (setext_h1_underline) @markup.heading.1.marker)
 
 (setext_heading
   (paragraph) @markup.heading.2
-  (setext_h2_underline) @markup.heading.2)
+  (setext_h2_underline) @markup.heading.2.marker)
 
 (atx_heading
-  (atx_h1_marker) @markup.heading.1
+  (atx_h1_marker) @markup.heading.1.marker
   (inline) @markup.heading.1)
 
 (atx_heading
-  (atx_h2_marker) @markup.heading.2
+  (atx_h2_marker) @markup.heading.2.marker
   (inline) @markup.heading.2)
 
 (atx_heading
-  (atx_h3_marker) @markup.heading.3
+  (atx_h3_marker) @markup.heading.3.marker
   (inline) @markup.heading.3)
 
 (atx_heading
-  (atx_h4_marker) @markup.heading.4
+  (atx_h4_marker) @markup.heading.4.marker
   (inline) @markup.heading.4)
 
 (atx_heading
-  (atx_h5_marker) @markup.heading.5
+  (atx_h5_marker) @markup.heading.5.marker
   (inline) @markup.heading.5)
 
 (atx_heading
-  (atx_h6_marker) @markup.heading.6
+  (atx_h6_marker) @markup.heading.6.marker
   (inline) @markup.heading.6)
 
 (info_string) @label
@@ -51,16 +51,16 @@
 (indented_code_block) @markup.raw.block
 
 ((fenced_code_block) @markup.raw.block
-  (#set! "priority" 90))
-
-(fenced_code_block
-  (fenced_code_block_delimiter) @markup.raw.block
-  (#set! conceal ""))
-
-(fenced_code_block
-  (info_string
-    (language) @label
-    (#set! conceal "")))
+  (#set! "priority" 15))
+
+; (fenced_code_block
+;   (fenced_code_block_delimiter) @markup.raw.delimiter
+;   (#set! conceal ""))
+;
+; (fenced_code_block
+;   (info_string
+;     (language) @conceal
+;     (#set! conceal "")))
 
 (link_destination) @markup.link.url
 
