diff --git a/queries/markdown/highlights.scm b/queries/markdown/highlights.scm
index 7c26fd71..cbaa8f27 100644
--- a/queries/markdown/highlights.scm
+++ b/queries/markdown/highlights.scm
@@ -51,16 +51,16 @@
 (indented_code_block) @markup.raw.block
 
 ((fenced_code_block) @markup.raw.block
-  (#set! "priority" 90))
-
-(fenced_code_block
-  (fenced_code_block_delimiter) @markup.raw.delimiter
-  (#set! conceal ""))
-
-(fenced_code_block
-  (info_string
-    (language) @conceal
-    (#set! conceal "")))
+  (#set! "priority" 15))
+
+; (fenced_code_block
+;   (fenced_code_block_delimiter) @markup.raw.delimiter
+;   (#set! conceal ""))
+;
+; (fenced_code_block
+;   (info_string
+;     (language) @conceal
+;     (#set! conceal "")))
 
 (link_destination) @markup.link.url
 
